app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -122,55 +122,48 @@ def self.default_blacklisted_event_names
 
   def self.params_for_create
     @params_for_create ||= {
-      :title  => "Configure AWS",
       :fields => [
         {
-          :component  => "text-field",
-          :name       => "name",
-          :label      => "Name",
-          :isRequired => true,
-          :validate   => [{:type => "required-validator"}]
-        },
-        {
-          :component  => "text-field",
-          :name       => "zone_name",
-          :label      => "Zone",
-          :isRequired => true,
-          :validate   => [{:type => "required-validator"}]
-        },
-        {
-          :component  => "text-field",
+          :component  => "select-field",
           :name       => "provider_region",
-          :label      => "Region",
+          :label      => _("Region"),
           :isRequired => true,
-          :validate   => [{:type => "required-validator"}]
+          :validate   => [{:type => "required-validator"}],
+          # TODO: we will need some kind of dynamic loading component that's able to load stuff based on static information
         },
         {
-          :component  => "text-field",
-          :name       => "endpoints.default.userid",
-          :label      => "Access Key",
-          :isRequired => true,
-          :validate   => [{:type => "required-validator"}]
+          :component => 'sub-form',
+          :name      => 'endpoints',
+          :title     => _("Endpoints"),
+          :fields    => [
+            {
+              :component => "text-field",
+              :name      => "<TBD>", # TODO: maybe proxy uri? (default_url in the old world)
+              :label     => _("Endpoint URL"),
+            },
+            {
+              :component => "text-field",
+              :name      => "endpoints.default.service_account",
+              :label     => _("Assume role ARN"),
+            },
+            {
+              :component  => "text-field",
+              :name       => "endpoints.default.userid",
+              :label      => _("Access Key ID"),
+              :isRequired => true,
+              :validate   => [{:type => "required-validator"}]
+            },
+            {
+              :component  => "text-field",
+              :name       => "endpoints.default.password",
+              :label      => _("Secret Access Key"),
+              :type       => "password",
+              :isRequired => true,
+              :validate   => [{:type => "required-validator"}]
+            },
+          ],
         },
-        {
-          :component  => "text-field",
-          :name       => "endpoints.default.password",
-          :label      => "Secret Key",
-          :type       => "password",
-          :isRequired => true,
-          :validate   => [{:type => "required-validator"}]
-        },
-        {
-          :component => "text-field",
-          :name      => "endpoints.default.proxy_uri",
-          :label     => "Proxy URI"
```

Comment:
Do we need this, is this the same as the `default_url` from the right? @agrare

Reply:
I believe that this is different from the default_url, we need to figure out when you add an amazon provider from the UI today with these fields filled in, where in the database do those fields go?

Reply:
As the old UI doesn't use this, I'm just omitting it for now.